__tests__/app.test.js

```diff
@@ -143,3 +143,62 @@ describe("GET /api/articles", () => {
 			});
 	});
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+	it("GET 200: responds with all comments that correspond to the given article ID'", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body: { comments } }) => {
+				expect(comments.length).toBe(2);
+				comments.forEach((comment) => {
+					expect(comment.article_id).toBe(3);
+				});
+			});
+	});
+	it("GET 200: comments contain the expected properties", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body: { comments } }) => {
+				comments.forEach((comment) => {
+					expect(comment).toEqual({
+						comment_id: expect.any(Number),
+						votes: expect.any(Number),
+						created_at: expect.any(String),
+						author: expect.any(String),
+						body: expect.any(String),
+						article_id: expect.any(Number),
+					});
+				});
+			});
+	});
+	it("GET 200: repsonds with most recent comments first (i.e. descending date order)", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body: { comments } }) => {
+				for (let i = 1; i < comments.length; i++) {
+					expect(Date.parse(comments[i - 1].created_at)).toBeGreaterThanOrEqual(
+						Date.parse(comments[i].created_at)
+					);
+				}
+			});
```

Comment:
We should add a test for when the article has no comments associated with it, like with article id 2 :)

Reply:
tested and passing :) (used a util function)

	it("GET 200: responds with an empty array when an existing article ID is used, but there are no comments associated with that article yet", () => {
		return request(app)
			.get("/api/articles/2/comments")
			.expect(200)
			.then(({ body: { comments } }) => {
				expect(comments).toEqual([]);
			});
	});